ssz-rs/benches/compute_proof.rs

```diff
@@ -0,0 +1,56 @@
+use criterion::{criterion_group, criterion_main, BenchmarkId, Criterion};
+use std::hint::black_box;
+
+use ssz_rs::{List, PathElement, Prove};
+use std::convert::TryFrom;
+
+/// Generate fake transactions to benchmark with.
+fn generate_dummy_transactions(size: usize) -> List<[u8; 32], 1000> {
```

Comment:
For the return type, if your goal is to test the generation of a transaction root you could use the following types (from the `ethereum_consensus` package):
```rs
/// 2**30 (= 1,073,741,824)
const MAX_BYTES_PER_TRANSACTION: usize = 1073741824;
/// 2**20 (= 1,048,576)
const MAX_TRANSACTIONS_PER_PAYLOAD: usize = 1048576;
pub type Transaction = ByteList<MAX_BYTES_PER_TRANSACTION>;
pub type Transactions = List<Transaction<MAX_BYTES_PER_TRANSACTION>, MAX_TRANSACTIONS_PER_PAYLOAD>,
```

Reply:
+1, where `ByteList<MAX_BYTES_PER_TRANSACTION>` = `List<u8, MAX_BYTES_PER_TRANSACTION>`